september/src/main/java/edu/asu/diging/gilesecosystem/september/web/HomeController.java

```diff
@@ -8,27 +8,60 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
 
+import edu.asu.diging.gilesecosystem.september.core.db.IMessageDbClient;
 import edu.asu.diging.gilesecosystem.september.core.model.IMessage;
+import edu.asu.diging.gilesecosystem.september.core.model.impl.DataTableData;
 import edu.asu.diging.gilesecosystem.september.core.service.IMessageManager;
 
 @Controller
 public class HomeController {
-    
-    @Autowired
-    private IMessageManager manager;
-
-    @RequestMapping(value = "/")
-    public String home(Principal principal, Model model, @RequestParam(defaultValue = "0") int page) {
-        if (principal == null || principal.getName() == null || principal.getName().equals("anonymousUser")) {
-            return "home";
-        }
-        
-        List<IMessage> messages = manager.getMessages(page);
-        model.addAttribute("messages", messages);
-        model.addAttribute("totalPages", manager.getNumberOfPages());
-        model.addAttribute("currentPageValue", page);
-        
-        return "home";
-    }
+
+	@Autowired
+	private IMessageManager manager;
+
+	@Autowired
+	private IMessageDbClient client;
+
+	@RequestMapping(value = "/")
+	public String home(Principal principal, Model model, @RequestParam(defaultValue = "0") int page) {
+		if (principal == null || principal.getName() == null || principal.getName().equals("anonymousUser")) {
+			return "home";
+		}
+
+		List<IMessage> messages = manager.getMessages(page);
+		model.addAttribute("messages", messages);
+		model.addAttribute("totalPages", manager.getNumberOfPages());
+		model.addAttribute("currentPageValue", page);
+		System.out.println("Messages Size: " + messages.size());
+		return "home";
+	}
+
+	@RequestMapping(value = "/datatable")
```

Comment:
really bad name; datatable of what, for what? the url should be an indicator for the data it provides;